rust/cardano-chain-follower/src/chain_sync.rs

```diff
@@ -529,7 +534,7 @@ pub(crate) async fn chain_sync(cfg: ChainSyncConfig, rx: mpsc::Receiver<MithrilU
     // Live Fill data starts at fork 1.
     // Immutable data from a mithril snapshot is fork 0.
     // Live backfill is always Fork 1.
-    let mut fork_count: u64 = 2;
+    let mut fork_count: Fork = 2.into();
```

Comment:
```suggestion
    let mut fork_count: Fork = Fork::FIRST_LIVE;
```

Suggest adding a constant for the first live fork (2) rather than using `2` directly.